metabase/metabase
# Use metabase-lib v2 to manage limit in the notebook editor

Epic #28034

Makes the notebook editor's `LimitStep` use metabase-lib v2 under the hood. Also, lays the foundation for porting other notebook steps to v2

### To Verify

1. Open `/question/:id/notebook`
2. Click the "Row limit" button to open the input
3. Try setting different row limit values, try different kind of queries, try multistage queries
4. Ensure everything works as on `master`


- - -
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/metabase/metabase/29618)
